src/managers/foodManager.ts

```diff
@@ -25,7 +30,7 @@ export async function getFood(search?: string, page?: string, limit?: string): P
 
   const food = await foodRepository.getFood(search, start, skip);
 
-  if (search) CheckIfSearchMatch(food, search);
+  if (search) CheckIfSearchMatch(food, search, user);
```

Comment:
Please remove the user from `CheckIfSearchMatch` function,  check here if (search && !user)